sidebar/view-callbacks.js

```diff
@@ -10,7 +10,8 @@ canViewCallbacks.attr('selected-in-sidebar', function(element) {
   // Using rAF because otherwise getBoundingClientRect won’t return useful values
   // Need to wait two animation frames for the correct scrollTop to be calculated
   requestAnimationFrame(function() {
-    requestAnimationFrame(function() {
+    // fixes bug after upgrade
+    setTimeout(function() {
```

Comment:
@chasenlehara I had to do this to get tests to pass.  When you have time, we should talk about why this works  this way.